### PR TITLE
feat(elements|ino-snackbar): add property for warning and error states

### LINKED ISSUE #115: Add color schemes to ino-snackbar
### Describe the solution you'd like
The user should be able to change the overall color scheme of the `ino-snackbar` component. Hence, I would like to add a new `inoColorScheme` property -  just like in the `ino-button` component - which can be set to either of the following color schemes: `primary`, `secondary`, `warning`, `error`

### PR DESCRIPTION
**Pull request by P1NHE4D**
✨ **Features:**
* Added a new `inoColorScheme` property to the `ino-snackbar` component

**Closes #115**

### CODE REVIEW

**Comment by janivo** [comment]
The new color schemes do not seem to match the designer's specifications. In the specification, it's only the border color that changes with each scheme.

**Comment by P1NHE4D** [comment]
> The new color schemes do not seem to match the designer's specifications. In the specification, it's only the border color that changes with each scheme.

whoops I overlooked the specifications - I'll fix it

File: packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const DefaultUsage = () => /*html*/ `
                 ino-message="${text('ino-message', sampleText)}"
                 ino-action-text="${text('ino-action-text', 'Anlegen')}"
                 ino-timeout="${number('ino-timeout', 5000)}"
+                ino-type="${select('ino-type', ['', 'error', 'warning'], '')}"
```
Comment (janivo) [inline packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.js:56]: Examples in the storybook for `warning` and `error` would be nice

**Review by janivo — CHANGES_REQUESTED** [review]

File: packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.js (hunk 1 above)
Comment (P1NHE4D) [reply packages/storybook/src/stories/ino-snackbar/ino-snackbar.stories.js:56]: ✅

File: packages/elements/src/components/ino-segment-button/readme.md [hunk 2]
```diff
@@ -67,18 +67,6 @@ class MyComponent extends Component {
 | `checkedChange` | Emits if the user interacts with the button. If the button is disabled or checked, the event will not be emitted. | `CustomEvent<any>` |
 
 
-## CSS Custom Properties
```
Comment (janivo) [inline packages/elements/src/components/ino-segment-button/readme.md:70]: This probably did not happen on purpose.

File: packages/elements/src/components/ino-snackbar/ino-snackbar.tsx [hunk 3]
```diff
@@ -29,6 +30,12 @@ export class Snackbar implements ComponentInterface {
    */
   @Prop() inoAlignment?: 'left' | 'right' | 'center' = 'center';
 
+  /**
+   * Changes the snackbar type
+   * Valid options are: warning, error
```
Comment (janivo) [inline packages/elements/src/components/ino-snackbar/ino-snackbar.tsx:35]: The options don't need to be mentioned explicitly. They will be generated

File: packages/elements/src/components/ino-snackbar/ino-snackbar.tsx [hunk 4]
```diff
@@ -29,6 +30,12 @@ export class Snackbar implements ComponentInterface {
...
+   */
+  @Prop() inoType?: SnackbarType;
```
Comment (janivo) [inline packages/elements/src/components/ino-snackbar/ino-snackbar.tsx:37]: If we use a literal type, we should provide a default value. Maybe add another option `default` which is the default value of `inoType` (analogous to line 31).

File: packages/elements/src/components/ino-snackbar/ino-snackbar.scss [hunk 5]
```diff
@@ -58,3 +60,11 @@ ino-snackbar {
     }
   }
 }
+
+@each $type in $snackbar-types {
+  ino-snackbar[ino-type="#{$type}"] {
```
Comment (janivo) [inline packages/elements/src/components/ino-snackbar/ino-snackbar.scss:65]: We should provide CSS vars to customize the color of these different types

**Review by janivo — CHANGES_REQUESTED** [review]

File: packages/elements/src/components/ino-snackbar/ino-snackbar.scss (hunk 5 above)
Comment (P1NHE4D) [reply packages/elements/src/components/ino-snackbar/ino-snackbar.scss:70]: ✅

File: packages/elements/src/components/ino-segment-button/readme.md (hunk 2 above)
Comment (P1NHE4D) [reply packages/elements/src/components/ino-segment-button/readme.md]: Nope - this happened by accident. Should be fixed now.

File: packages/elements/src/components/ino-snackbar/ino-snackbar.tsx (hunk 4 above)
Comment (P1NHE4D) [reply packages/elements/src/components/ino-snackbar/ino-snackbar.tsx]: ✅

**Review by janivo — APPROVED** [review]

**Pull request merged**